bbtest/hosts.py

```diff
@@ -33,41 +39,53 @@ def __init__(self, *args, **kwargs):
         except AttributeError:
             self.root_path = tempfile.gettempdir()
 
+    def install(self):
+        """ install host for bbtest
+
+        We separate install from init so we can create an install hosts in different order.
+        """
+        pass
+
+    def uninstall(self):
+        """ the opposite of install,  uninstall bbtest from host """
+        pass
+
+    def clean(self):
+        raise NotImplementedError('Missing method implementation')
+
     @property
     def os(self):
-        raise NotImplementedError('Missing method implmentation')
+        raise NotImplementedError('Missing method implementation')
 
     @property
     def bit(self):
-        raise NotImplementedError('Missing method implmentation')
+        return platform.machine()[-2:]
+
+    @property
+    def name(self):
+        return socket.gethostname()
```

Comment:
These two functions belong in `LocalHost`

Reply:
Done